Makefile

```diff
@@ -356,12 +356,17 @@ chglog:
 docs:
 	@$(MAKE) docs\:cli
 	@$(MAKE) docs\:manpages
+	@$(MAKE) docs\:http
 
 .PHONY: docs\:cli
 docs\:cli:
 	rm -f docs/website/references/cli/*.md
 	go run cmd/genclidocs/main.go -o docs/website/references/cli
 
+.PHONY: docs\:http
+docs\:http:
+	go run cmd/genopenapi/main.go | python -m json.tool > docs/website/references/http/openapi.json
```

Comment:
question: Why do we want to have the openAPI json output published in our docs? I'm asking because it's not really something to be read by humans.

Reply:
we can have it render a UI like this with some config options https://docs.source.network/sourcehub/api

Reply:
I see. Hopefully we can make the table easier to read because the one at the link is pretty terrible 😅

Reply:
This is one of my favourite API docs: https://docs.stripe.com/api 
If we can eventually make it look like this but on top of it be able to execute the request to a configurable endpoint, that would be amazing.

Reply:
> This is one of my favourite API docs: [docs.stripe.com/api](https://docs.stripe.com/api) If we can eventually make it look like this but on top of it be able to execute the request to a configurable endpoint, that would be amazing.

Nice the stripe docs look really nice. Whoever is responsible for website doc stuff can easily plug it in using the `json` file this PR introduces